fixed issue #5, implemented issue #8

itelo opened the pull request:
Caso nao tenha o arquivo .notes, o webnotes.py se encarrega de cria-lo.
Foi adicionado o suporte para criacao de notas pelo webnotes.py.

itelo committed:
fixed issue #5, implemented issue #8

Caso nao tenha o arquivo .notes, o webnotes.py se encarrega de cria-lo.
Foi adicionado o suporte para criacao de notas pelo webnotes.py.

itelo committed:
fixed issue #5, implemented issue #8

Caso nao tenha o arquivo .notes, o webnotes.py se encarrega de cria-lo.
Foi adicionado o suporte para criacao de notas pelo webnotes.py.

pedosb commented on webnote.py:
```diff
@@ -1,95 +1,115 @@
 s.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
 
-s.bind(('0.0.0.0', 80))
+s.bind(('0.0.0.0', 3000))
```
Não deves mudar a porta em um commit

pedosb commented on webnote.py:
```diff
@@ -1,95 +1,115 @@
+            field_body = field.group(2)
+            request_headers[field_name] = field_body
+            data = field.group(3)
+            if field_name == "Content-Length":
```
Desnecessário, não precisa colocar isso em uma variável, esse dado já está no dicionário (foi incluído na linha 46), para acessar basta fazer `request_headers['Content-Lenght']`.

pedosb commented on webnote.py:
```diff
@@ -1,95 +1,115 @@
+        else:
+            raise Exception('Cannot read request header')
+            
+    print resource
```
Não deverias debugar com `print`, muito menos colocá-lo em um commit

pedosb commented on webnote.py:
```diff
@@ -1,95 +1,115 @@
+            raise Exception('Cannot read request header')
+            
+    print resource
+    print request_headers
```
Não deverias debugar com `print`, muito menos colocá-lo em um commit

pedosb commented on webnote.py:
```diff
@@ -1,95 +1,115 @@
+                rl_body_len = len(data)
+                if rl_body_len != body_len:
+                    new_data = conn.recv(1024)
+                    data += new_data
```
Se não vais usar o `new_data` pra nada pode fazer direto a concatenação com `data += conn.recv(1024)`, assim como o `r1_bod_len`, podes colocar o `len(data)` dentro do `if`.

pedosb commented:
Da uma olhada nesses comentários e ai incorporarei teu merge :)

itelo committed:
fixed issue #5, implemented issue #8

Caso nao tenha o arquivo .notes, o webnotes.py se encarrega de cria-lo.
Foi adicionado o suporte para criacao de notas pelo webnotes.py.
